scheduler/reconcile_util.go

```diff
@@ -313,6 +326,10 @@ func (a allocSet) filterByTainted(taintedNodes map[string]*structs.Node, support
 					lost[alloc.ID] = alloc
```

Comment:
This case above has the same result but it's not gated behind `supportsDisconnectedClients` like the disconnecting case above it. Should this line require support for disconnected clients too?

Reply:
Yes!  It should.  Good catch.

Reply:
Done